packages/application-shell/src/components/navbar/menu-items.tsx

```diff
@@ -270,22 +252,23 @@ type MenuItemProps = {
 };
 const MenuItem = (props: MenuItemProps) => {
   return (
-    <li
+    <MenuListItem
       role="menuitem"
       onClick={props.onClick}
       onMouseEnter={props.onMouseEnter as MouseEventHandler<HTMLElement>}
       onMouseLeave={props.onMouseLeave as MouseEventHandler<HTMLElement>}
       onFocus={props.onMouseEnter as FocusEventHandler<HTMLElement>}
       onBlur={props.onMouseLeave as FocusEventHandler<HTMLElement>}
       data-menuitem={props.identifier}
-      className={classnames('list-item', {
-        item__active: props.isActive,
-        item_menu__active: props.isMainMenuRouteActive ?? false,
-        'item_menu-collapsed': !props.isMenuOpen,
+      className={classnames({
+        active: props.isActive,
```

Comment:
classnames are retained here and in the  [`<MenuList>`](https://github.com/commercetools/merchant-center-application-kit/blob/a6ea9671704ccb3e40690b8df2c7ef7046985267/packages/application-shell/src/components/navbar/menu-items.tsx#L203-L220) rendered as part of  `<MenuGroup>` since some styling changes are applied from the outside of those components e.g. [here](https://github.com/commercetools/merchant-center-application-kit/blob/a6ea9671704ccb3e40690b8df2c7ef7046985267/packages/application-shell/src/components/navbar/main-navbar.styles.ts#L88) and [here](https://github.com/commercetools/merchant-center-application-kit/blob/a6ea9671704ccb3e40690b8df2c7ef7046985267/packages/application-shell/src/components/navbar/menu-items.styles.ts#L285-L317)